src/api/neoscan.js

```diff
@@ -76,16 +75,33 @@ export const getBalance = (net, address) => {
  */
 export const getClaims = (net, address) => {
   const apiEndpoint = getAPIEndpoint(net)
-  return axios.get(apiEndpoint + '/v1/get_claimable/' + address)
-    .then((res) => {
-      const claims = parseClaims(res.data.claimable)
-      log.info(`Retrieved Balance for ${address} from neoscan ${net}`)
-      return new Claims({ net, address: res.data.address, claims })
-    })
+  return axios.get(apiEndpoint + '/v1/get_claimable/' + address).then(res => {
+    const claims = parseClaims(res.data.claimable)
+    log.info(`Retrieved Balance for ${address} from neoscan ${net}`)
+    return new Claims({ net, address: res.data.address, claims })
+  })
+}
+
+/**
+ * Get spent and unspent claimable amounts for an address.
+ * @param {string} net - 'MainNet', 'TestNet' or a custom NeoScan-like url.
+ * @param {string} address - Address to check.
+ * @return {Promise<Claim>}
+ */
+export const getClaimBalance = (net, address) => {
+  const apiEndpoint = getAPIEndpoint(net)
+  return axios.get(apiEndpoint + '/v1/get_claimable/' + address).then(res => {
+    let claimBalance = 0

```

Comment:
let claimBalance = new Fixed8(0)
if (res.data.unclaimed) claimBalance = new Fixed8(res.data.unclaimed)

wheres the addition?

anyway to add Fixed8:

```
claimBalance = claimBalance.add(whateveryouwanttoadd)
```